### PR TITLE
[EXPORTER] bump prometheus to v1.3.0

### PR DESCRIPTION
**Pull request by esigo**
Fixes # (issue)

## Changes
bumps prometheus from `1.2.4` to `v1.3.0`.

For significant contributions please make sure you have completed the following items:

* [x] `CHANGELOG.md` updated for non-trivial changes
* [ ] Unit tests have been added
* [ ] Changes in public API reviewed

### CODE REVIEW

**Review by marcalff — APPROVED** [review]
LGTM, thanks for the upgrade.

Totally optional, please consider to add a section in file
`docs/maintaining-dependencies.md` as well, for prometheus-cpp.

**Review by ThomsonTan — APPROVED** [review]

**Comment by esigo** [comment]
@marcalff thanks, added a section in file `docs/maintaining-dependencies.md`, for prometheus-cpp.

File: docs/maintaining-dependencies.md [hunk 1]
```diff
@@ -442,3 +442,155 @@ Last, in some special case like name collisions for a given symbol,
 the template itself may need to be adjusted for special logic.
 
 See for example how `messaging.client_id` is treated.
+
+## prometheus-cpp
+
+### Comments (prometheus-cpp)
+
+The `prometheus-cpp` library provides a C++ client for Prometheus, facilitating the creation and registration of metrics that Prometheus scrapes. `prometheus-cpp` is used as a git submodule under the `third_party` directory for ease of inclusion in build system.
```
Comment (marcalff) [inline docs/maintaining-dependencies.md:450]:
```suggestion
The `prometheus-cpp` library provides a C++ client for Prometheus,
facilitating the creation and registration of metrics that Prometheus scrapes.
`prometheus-cpp` is used as a git submodule under the `third_party` directory
for ease of inclusion in build system.
```

File: docs/maintaining-dependencies.md [hunk 2]
```diff
@@ -442,3 +442,155 @@ Last, in some special case like name collisions for a given symbol,
...
+
+### Origin (prometheus-cpp)
+
+The repository for `prometheus-cpp` can be found here:
+
+- [repository](https://github.com/jupp0r/prometheus-cpp)
+
+Check release notes at:
+
+- [release-notes](https://github.com/jupp0r/prometheus-cpp/releases)
```
Comment (marcalff) [inline docs/maintaining-dependencies.md:460]:
```suggestion
* [release-notes](https://github.com/jupp0r/prometheus-cpp/releases)
```

File: docs/maintaining-dependencies.md [hunk 3]
```diff
@@ -442,3 +442,155 @@ Last, in some special case like name collisions for a given symbol,
...
+
+### Comments (prometheus-cpp)
+
+The `prometheus-cpp` library provides a C++ client for Prometheus, facilitating the creation and registration of metrics that Prometheus scrapes. `prometheus-cpp` is used as a git submodule under the `third_party` directory for ease of inclusion in build system.
+
+### Origin (prometheus-cpp)
+
+The repository for `prometheus-cpp` can be found here:
+
+- [repository](https://github.com/jupp0r/prometheus-cpp)
```
Comment (marcalff) [inline docs/maintaining-dependencies.md:456]:
```suggestion
* [repository](https://github.com/jupp0r/prometheus-cpp)
```

**Comment by keith** [comment]
for future reference looks like this missed the MODULE.bazel file, but maybe it wasn't in the central registry at the time either, submitted: https://github.com/open-telemetry/opentelemetry-cpp/pull/3162

**Pull request merged**